deployment/promotion/promote.py

```diff
@@ -183,12 +182,24 @@ def upload_artifact(auth, model_version_id, artifact):
     key = artifact['key']
     print("Uploading artifact '%s'" % key)
 
-    upload_url = get_artifact_url(auth, model_version_id, artifact)
+    put_url = get_artifact_url(auth, model_version_id, artifact)
```

Comment:
confuzzled why you're calling it `put_url` when it's clearly a get

Reply:
It is used below in the call to `requests.put`, the `get_artifact_url` doesn't refer to the HTTP verb, but that we are fetching an S3 pre-signed URL from the Deploy API

Reply:
well it's obviously not a get, it's a put (line 187)

Reply:
(`artifact` here is really a map which contains `method` and `key`)

Reply:
Renamed to `signed_artifact_url`